docs-chef-io/content/supermarket/ctl_supermarket.md

```diff
@@ -29,6 +29,43 @@ sudo -u supermarket supermarket-ctl make-admin USER_NAME
 where `USER_NAME` represents the name of the user to be granted
 administrator privileges.
 
+## SPDX license update
+
```

Comment:
We should make sure to document the purpose of the SPDX commands. Why does Supermarket include SPDX data and why would a person ever need to run these commands?

Reply:
That we have already covered as part of this PR - https://github.com/chef/supermarket/pull/2296/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5 here -- actually we missed SPDX command documentation in earlier documentation and hence raised this PR separately

Reply:
The readme isn't for users though. The readme is for developers. The docs site is where all user facing documentation should live.

Reply:
ohh ok .. I will make the required changes in documentation here